util/options.py

```diff
@@ -1,31 +1,186 @@
 import json
+import logging
 import os
+import subprocess
 
 import yaml
+from pydantic import BaseModel, SecretStr, constr
+from pydantic_settings import BaseSettings
 
+logger = logging.getLogger(__name__)
 
-class Options:
```

Comment:
Given the removal of the options class and creation of `Settings` would it be appropriate to rename this entire file to `settings.py`?